core/consensus/babe/impl/babe_impl.cpp

```diff
@@ -181,7 +190,30 @@ namespace kagome::consensus::babe {
 
     // won't start block production without keypair
     if (not keypair_) {
+      // launch timer that would check key appearance
+      if (not key_wait_ticker_->isStarted()) {
+        key_wait_ticker_->asyncCallRepeatedly(
+            [wp = weak_from_this()](auto &&ec) {
+              if (auto self = wp.lock()) {
+                if (ec) {
```

Comment:
I guess, case for cancel in not error here

Reply:
can't find any mention of cancel error

Reply:
It is `boost::system::errc::operation_canceled` error code